var/spack/repos/builtin/packages/flann/package.py

```diff
@@ -0,0 +1,134 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+
+class Flann(CMakePackage):
+    """
+    FLANN is a library for performing fast approximate nearest neighbor
```

Comment:
Technically it's a valid docstring. The problem is that we generate a [Package List](http://spack.readthedocs.io/en/latest/package_list.html) based on these docstrings. If you run `spack list --format=rst`, you'll see what it looks like. If you take the last package in the list (`zstd`) and add a newline to the start of the docstring, you'll see that it prints:
```rst
Description:
   Zstandard, or zstd as short version, is a fast lossless compression
  algorithm, targeting real-time compression scenarios at zlib-level and
  better compression ratios.
```
The newline causes the indentation to be off by one space. I would have to look at the code that does this. Perhaps we could strip leading and trailing newline characters from the docstring before printing.